lib/scenic/adapters/postgres.rb

```diff
@@ -46,6 +48,16 @@ def create_view(name, sql_definition)
         execute "CREATE VIEW #{name} AS #{sql_definition};"
       end
 
+      # Updates a view in the database.
```

Comment:
Since it's the PG adapter, I think it could be interesting to explain the internals a bit to help spread some knowledge you and I have but others might not. Proposal:

> Postgres does not allow columns to be removed or reordered from a view in an `CREATE OR REPLACE VIEW` statement. Because we'd like to allow this in update_view, we drop the view and re-create it using the new definition. See http://www.postgresql.org/docs/9.4/static/sql-createview.html for more.